linode/lke/cluster.go

```diff
@@ -338,6 +338,14 @@ func waitGroupCh(wg *sync.WaitGroup) <-chan struct{} {
 	return done
 }
 
+func poolMatchesDeclaredPool(pool *linodego.LKEClusterPool, declaredPool map[string]interface{}) bool {
+	if pool.Type != declaredPool["type"] || pool.Count != declaredPool["count"] {
+		return false
+	}
+
+	return true
+}
```

Comment:
Just a small stylistic change :+1: 
```suggestion
return pool.Type == declaredPool["type"] && pool.Count == declaredPool["count"]
```